.gitignore

```diff
@@ -4,4 +4,5 @@
 *.sh
 *.ini
 *.pem
-site.yml
\ No newline at end of file
+site.yml
+tests/test.retry
```

Comment:
Missing new line

Reply:
It's my IDE, I have the linter running now and will fix those missing lines :)